### DISCUSSION TITLE
Restrict Service Access with Mesh - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3735] [Announcements]
# Restrict Service Access with Mesh - DaoCloud Enterprise

DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote and boost your digital business.

https://docs.daocloud.io/en/mspider/best-practice/use-egress-and-authorized-policy.html

**Comment by hanxiaop** [comment]
中英文图片混用了

**Reply by windsonsea** [reply]
updated those images in PR #5325
